src/Jackett.Common/Definitions/letseed.yml

```
@@ -0,0 +1,240 @@
+    selector: img#regimage
+    input: imagestring
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    logout: ""
+  error:
+    - selector: >-
+        #main table:has(td:contains("Bir hata oluştu"))
+    # Sometimes the login form POST gets a proper `302.Redirect`, but sometimes it gets
+    # a `http-equiv="refresh"` "redirect" instead:
+    - selector: >-
+        [http-equiv="refresh"][content*="/login.php?error=1"]
+      message:
+        text: >-
+          HATA: Geçersiz kullanıcı adı veya şifre! Lütfen tekrar eneyin veya
+          buraya tıklayarak şifrenizi sıfırlayınız.
+    - selector: >-
+        [http-equiv="refresh"][content*="/login.php?error=2"]
+      message:
+        text: >-
+          HATA: Girdiğiniz resim doğrulama kodu onaylanmadı.
+    - selector: >-
+        [http-equiv="refresh"][content*="/login.php?error=3"]
+      message:
+        text: >-
+          HATA: Lütfen gerekli alanları boş bırakmayınız!
+    - selector: >-
+        [http-equiv="refresh"][content*="/login.php?error=4"]
+      message:
+        text: >-
+          HATA: Kullanıcı adı veya şifre hatalı!
+    - selector: >-
+        [http-equiv="refresh"][content*="/login.php?error="]
+      message:
+        text: >-
+          HATA: Bir hata oluştu!
+  test:
+    path: browse.php
+    selector: a[href*="/logout.php"]
+    # TODO: While debugging false login success for this definition, I read in the docs
+    # that a redirect is considered as failure and that `selector:` is only required for
+    # web UIs that render the login form directly in the response to `path:`. For all
+    # the previous definitions I've worked on *and* the examples I copied for them, the
+    # web UIs responded with redirects but the definitions include `selector:`. Are
+    # those existing definitons wrong in this aspect?  If no, why use `selector:` for
+    # web UIs that redirect to the login form?
+
+download:
+  before:
+    path: takethanks.php
+    method: post
+    inputs:
+      torrentid: >-
+        {{ re_replace .DownloadUri.AbsolutePath "/download-torrent-([0-9]+).*" "$1"}}
+search:
+  paths:
+    - path: browse.php
+      method: post
+  keywordsfilters:
+    - name: re_replace
+      args: ["[\\s]+", "%"]
+  inputs:
+    do: search
+    keywords: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
+    # t_name, t_description, t_both, t_uploader, t_genre
+    search_type: "{{ if .Query.IMDBID }}t_genre{{ else }}t_name{{ end }}"
+    # does not support multi category searching so defaulting to all.
+    category: 0
+    # yes, no
+    include_dead_torrents: yes
+    sort: "{{ if .Config.freeleech }}free{{ else }}{{ .Config.sort }}{{ end }}"
+    order: "{{ if .Config.freeleech }}asc{{ else }}{{ .Config.type }}{{ end }}"
+
+  rows:
+    selector: "table.sortable tr:has(a[href*=\"/download-torrent-\"]){{ if .Config.freeleech }}:has(img[src$=\"/freedownload.gif\"]){{ else }}{{ end }}"
```

```suggestion
    selector: table.sortable tr:has(a[href*="/download-torrent-"])
```

If we're using `t_genre` to search IMDB ID, then we don't need to also filter results.